SMA_FluxExtraction.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>When a source is found to be partially resolved, it would be useful to clarify where to find the tutorial on the ensuing process steps.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>